VizDotLife

```diff
@@ -0,0 +1,30 @@
+{
+    "project": "Viz Dot Life",
+    "tags": [
+        "Viz.Life",
+        "Viz Dot Life",
+        "VizDotLife",
+        "www.viz.life",
```

Comment:
remove comma from this line

Reply:
Thank you. I made the change and submitted a new Pull Request a few minutes ago